### DISCUSSION TITLE
Is there anyway to strip the binary after building

### DISCUSSION THREAD

**Discussion by tigerinus** [goreleaser/goreleaser#3291] [Q&A] [answered]
Currently binaries built are not [stripped](https://en.wikipedia.org/wiki/Strip_%28Unix%29).

I wonder if there is any trick to do this or is it a nice to have feature for future goreleaser?

Thanks.

**Comment by caarlos0** [comment, marked as answer]
probably doable using build hooks

https://goreleaser.com/customization/build/
